toDo/scripts/index.js

```diff
@@ -0,0 +1,59 @@
+let todoControl = document.querySelector('.todo-control');
+let headerInput = document.querySelector('.header-input');
+let todoList = document.querySelector('.todo-list');
+let todoCompleted = document.querySelector('.todo-completed');
+let btnAddTask = document.querySelector('.header-button');
+let todoData = [];
+
+let render = function () {
+  todoList.textContent = '';
+  todoCompleted.textContent = '';
+  todoData.forEach((item, index) => {
+    let li = document.createElement('li');
+    li.classList.add('todo-item', 'todo-item_' + index);
+    li.innerHTML = '<span class="text-todo">' + item.value + '</span>' +
+      '<div class="todo-buttons">' + '<button class="todo-remove"></button>' +
+      '<button class="todo-complete"></button>' + '</div>';
+    if (item.completed) {
+      todoCompleted.append(li);
+    } else {
+      todoList.append(li);
+    }
+    let BtnTodoComplete = li.querySelector('.todo-complete');
+    let BtnTodoRemove = li.querySelector('.todo-remove');
```

Comment:
```suggestion
    let btnTodoComplete = li.querySelector('.todo-complete');
    let btnTodoRemove = li.querySelector('.todo-remove');
```
с маленькой буквы нужно задавать названия переменных